src/connection-manager/manifests.ts

```diff
@@ -84,55 +80,24 @@ export async function handleManifest(
  */
 export async function updateUNLManifests(): Promise<void> {
   try {
-    log.info('Fetching UNL...')
-    const unl: UNLBlob = await fetchValidatorList(await getFirstUNL('main'))
-    const promises: Array<Promise<void>> = []
+    log.info('Fetching Manifests from UNLs...')
+    const networks = await getNetworks()
 
-    unl.validators.forEach((validator: UNLValidator) => {
-      const manifestHex = Buffer.from(validator.manifest, 'base64')
-        .toString('hex')
-        .toUpperCase()
-      promises.push(handleManifest(manifestHex))
+    await async.map(networks, async (network: Network) => {
```

Comment:
What does `async` do here?

Reply:
`async` would be to ensure all inner promises got executed before the function returns, since this function incurs multiple loop layers of async execution. It could be done as well with multiple Promise.all layers but it would raise eslint issues and also returns a `Promise<void[][]>`. @ckniffen and I think it would be cleaner this way using async package.